charts/ceph-csi-cephfs/values.yaml

```diff
@@ -161,7 +161,7 @@ provisioner:
   provisioner:
     image:
       repository: k8s.gcr.io/sig-storage/csi-provisioner
-      tag: v2.2.2
+      tag: v3.0.0
```

Comment:
node-driver-registrar is not updated for cephfs

Reply:
Thanks @Madhu-1 , updated .